code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -802,11 +802,46 @@
 						continue
 					M.reagents.add_reagent("jenkem", 1)
 
+		if("hem")
+			message = "<b>[src]</b> hems."
+
+		if("spin")
+			spin(20, 1)
+			message = "<b>[src]</b> spins around dizzily!"
+
+		if("highfive")
+			if(restrained())
+				return
+			if(highfiving)
+				to_chat(src, "You give up on the high-five.")
+				highfiving = FALSE
+				return
+			highfiving = TRUE
+			for(var/mob/living/carbon/C in orange(1))
+				if(C.highfiving)
+					if(C.mind.special_role == "Wizard" && mind.special_role == "Wizard")
```

Comment:
Without apprentices you'll never see it, *shrug

And if anything, they're unwieldy power should be able to give off explosions... it just might not also protect themselves from that explosion ;)